Prepare bam 21.04.1 (#1257)
centreon/centreon-documentation: en/releases/centreon-commercial-extensions.md

## Centreon BAM

### 21.04.1

> `January 7, 2022`

Reviewer: ```suggestion
`January 21, 2022`
```

#### Improvements

- Add Feature Flipping for Resources Status vs Legacy Pages

> - ACL: Authorize visualisation on daughter Business Activities even if not part of the same Business View

Reviewer 2: ```suggestion
- ACL: Authorize visualisation on child Business Activities even if not part of the same Business View
```

- Add warning when enabling notifications for a BA, for the user to know about the necessity of setting notification commands

#### Bugfixes

> - Fixed the message in popup when deleting "Activités Métiers" (Fr language)

Reviewer 2: not totally clear

> - Fixed the wrong message in popup when deleting "Activités Métiers" (Fr language)

Reviewer 2: ```suggestion
- Fixed the message that appeared in a popup when deleting "Activités Métiers" (Fr language)
```

- Fixed an error when generating poller configuration with a disabled BA linked to an escalation task

> - Fixed the relations between BA and escalations are exported in RS engine conf regardless the BA is linked to the RS or not

Reviewer 2: there are 2 verbs in the sentence

> - Fixed the relations between BA and escalations exported in RS engine conf regardless if the BA is linked to the RS or not

Reviewer 2: ```suggestion
- Fixed the relations between BA and escalations exported in RS engine conf regardless of whether the BA is linked to the RS or not
```

> - Fixed the alignment of "required" input text on panel in Add Business Activity panel

Reviewer 2: in panel + in panel?

> - Fixed MBI menu entry not visible after upgrade from 19.10 to 21.10.0-beta.2 due to BAM module

Reviewer 2: ```suggestion
- Fixed MBI menu entry that was not visible after upgrade from 19.10 to 21.10.0-beta.2 due to BAM module
```

> - Fixed MBI menu entry is not visible after upgrade from 19.10 to 21.10.0-beta.2 due to BAM module

Reviewer 2: 2 verbs
